lib/beaker-hostgenerator/data.rb

```diff
@@ -270,6 +276,11 @@ def osinfo
             'platform'           => 'debian-7-amd64',
             'packaging_platform' => 'debian-7-amd64'
           },
+          :docker => {
+            'docker_image_commands': [
+              'apt-get update && apt-get install -y cron locales-all net-tools wget'
```

Comment:
@ekohl Looks like a typo here where ':' should actually be '=>'

Reply:
Fixed